src/Catty/ViewController/Stage/StagePresenterSideMenuView.swift

```diff
@@ -144,42 +148,36 @@ enum SideMenuButtonType {
         }
 
         let backLabel = setupLabel(title: kLocalizedBack, selector: #selector(delegate?.stopAction), type: .landscapeFirstColumn)
-        backLabel.topAnchor.constraint(equalTo: backButton.bottomAnchor, constant: marginLabel).isActive = true
+        backLabel.topAnchor.constraint(equalTo: backButton.bottomAnchor).isActive = true
 
-        let continueButton = setupButton(imageName: "stage_dialog_button_continue", selector: #selector(delegate?.continueAction), type: .landscapeSecondColumn)
-        if #available(iOS 11.0, *) {
-            continueButton.topAnchor.constraint(equalTo: self.safeAreaLayoutGuide.topAnchor, constant: marginTopBottom).isActive = true
-        } else {
-            continueButton.topAnchor.constraint(equalTo: self.topAnchor, constant: marginTopBottom).isActive = true
-        }
+        let continueButton = setupButton(imageName: "stage_dialog_button_continue", selector: #selector(delegate?.continueAction), type: .landscapeFirstColumn)
+        continueButton.topAnchor.constraint(equalTo: backLabel.bottomAnchor, constant: marginTopBottom * 2).isActive = true
 
-        let continueLabel = setupLabel(title: kLocalizedContinue, selector: #selector(delegate?.continueAction), type: .landscapeSecondColumn)
-        continueLabel.topAnchor.constraint(equalTo: continueButton.bottomAnchor, constant: marginLabel).isActive = true
+        let continueLabel = setupLabel(title: kLocalizedContinue, selector: #selector(delegate?.continueAction), type: .landscapeFirstColumn)
+        continueLabel.topAnchor.constraint(equalTo: continueButton.bottomAnchor).isActive = true
+        continueLabel.bottomAnchor.constraint(equalTo: self.centerYAnchor).isActive = true
 
         let restartButton = setupButton(imageName: "stage_dialog_button_restart", selector: #selector(delegate?.restartAction), type: .landscapeFirstColumn)
-        restartButton.centerYAnchor.constraint(equalTo: self.centerYAnchor, constant: (marginTopBottom - marginLabel) * -1).isActive = true
+        restartButton.topAnchor.constraint(equalTo: continueLabel.bottomAnchor, constant: marginTopBottom * 2).isActive = true
```

Comment:
I dont understand why you use marginTopBottom * 2.
For me it seems better to set the RestartButton.TopAncor to the Mid. So that Continue Label is over the Mid and Restart Button under the Mid